docs/DeveloperGuide.md

```diff
@@ -492,3 +501,68 @@ testers are expected to do more *exploratory* testing.
     1. _{ tbc }_
 
 1. _{ more test cases …​ }_
+
+## **Appendix: Planned Enhancements**
+
+Team size: 5
+
+1. **Enable cross day free time interval:**
+   * Currently, a free time interval can only be specified based on a single day. There is no way to specify a free time interval that spans multiple day.
+   * To improve user experience, it could be useful to enable cross day free time interval, such as `Mon:2000-Tue:0800`.
+
+2. **Enable free time interval merging:**
+   * Currently, a person can have overlapping free time intervals.
+     * For example, a person can have free time intervals of `Mon:1000-1200` and `Mon:1100-1300`, or `Mon:2000-2359` and `Tue:0000-0800`.
+   * To improve user experience, it could be useful to merge free time tags that have overlapping time intervals.
+     * For example, `Mon:1000-1200` and `Mon:1100-1300` can be combined as `Mon:1000-1300`.
+     * For example, `Mon:2000-2359` and `Tue:0000-0800` can be combined as `Mon:2000-Tue:0800`.
+   * The merging process could be done when new free time tag is added to the person, e.g. via `add` command, `addTime` command, or `edit` command.
+
+3. **Delete free time within an interval:**
+   * Additionally, it could be useful to be able to delete free times that fall within a time interval, e.g. via `deleteTime` command.
+   * For example, an initial freeTimeTag could be `Mon:1000-1400`. Executing `deleteTime INDEX ft/Mon:1100-1200` will result in the new freeTimeTags `Mon:1000-1100` and `Mon:1200-1400`.
+
+4. **Allow more flexibility in command prefixes:**
```

Comment:
I think need to specify Make command prefixes case-insensitive.

Reply:
Will change in the next update.